MyFTP/ProgramServer/Server.cs

```diff
@@ -0,0 +1,111 @@
+namespace ProgramServer;
+
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+
+/// <summary>
+/// Class for MyFTP Server
+/// </summary>
+public class Server
+{
+    private readonly List<Task> _requests = new();
+    private readonly CancellationTokenSource _tokenSource = new();
+    private readonly TcpListener _server;
+     
+    /// <summary>
+    /// Constructor of server
+    /// </summary>
+    public Server(IPAddress ip, int port)
+    {
+        _server = new TcpListener(ip, port);
+    }
+    
+    /// <summary>       
+    /// Returns files and directories from server to client
+    /// </summary>
+    private async Task List(string path, StreamWriter writer)
+    {
+        if (!Directory.Exists(path))
+        {
+            await writer.WriteLineAsync("-1");
+            return;
+        }
+        var files = Directory.GetFiles(path);
+        var directories = Directory.GetDirectories(path);
+        var response = new StringBuilder((files.Length + directories.Length).ToString());
+        foreach (var file in files)
+        {
+            response.Append(" " + file + " false");
```

Comment:
Используйте string interpolation, 
```suggestion
            response.Append($" {file} false");
```